lib/createFormatter.js

```diff
@@ -8,10 +10,18 @@ module.exports = function createFormatter(newrelic) {
     if (info.exception === true) {
       // Due to Winston internals sometimes the error on the info object is a string or an
       // empty object, and so the message property is all we have
-      info['error.message'] = info.error.message || info.message || ''
+      const errorMessage = info.error.message || info.message || ''
+
+      info['error.message'] = truncate(errorMessage)
       info['error.class'] = info.error.name === "Error" ?
         info.error.constructor.name : info.error.name
-      info['error.stack'] = info.error.stack.substring(0, 1024)
+      info['error.stack'] = truncate(info.error.stack)
+      info.message = truncate(info.message)
```

Comment:
Reading your readme made me realize we might want to consider truncating message regardless of if there's an error. I suppose we can wait to see if there's an issue in the wild or do with a separate PR.